WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -15805,6 +15828,7 @@ e_armenia = {
 				OR = {
 					ai = no
 					religion_group = christian
+					culture = armenian
```

Comment:
"AI = NO" shoudn't be there. AI Armenians who manage the feat of breaking off from the Persians should be allowed to restore their empire. ANYONE SHOULD be able to do so, but those stupid restrictions are there. I only made them more berable, even though they shouldn't exist at all.